cost-analyzer/templates/etl-utils-deployment.yaml

```diff
@@ -35,6 +35,7 @@ spec:
             defaultMode: 420
             secretName: {{ .Values.kubecostModel.federatedStorageConfigSecret }}
         {{- end }}
+      serviceAccountName: {{ template "aggregator.serviceAccountName" . }}
```

Comment:
there could be circumstances where the etl utils could be deployed and the aggregator could not, right? maybe the cost analyzer SA template, and also, we provide a means to give this its own SA?

Reply:
good point!
one moment.